.ci_support/migrations/xz_to_liblzma_devel.yaml

```diff
@@ -0,0 +1,11 @@
+migrator_ts: 1733863935
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+
+# This migration should also rename xz in host to liblzma-devel
```

Comment:
Hi @h-vetinari,

I am trying to understand this comment.

Must or might this be done manually?

Reply:
Nevermind, ArcticDB does not depend on `xz`, for some reason I thought it did.

Reply:
The bot should do this automatically; I guess it didn't happen because there was no `- xz` dependency in the recipe.

Reply:
OK. The bot has not opened PRs on other branches for this migration. I'll do them manually tomorrow.